src/main/webapp/app/components/tokenInputGroups/TokenExtendButton.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { ReactNode } from 'react';
+import {
+  InputGroup,
+  Button,
+  Dropdown,
+  DropdownButton,
+  Row
+} from 'react-bootstrap';
+import classnames from 'classnames';
+import { observable, action } from 'mobx';
+import { observer } from 'mobx-react';
+import autobind from 'autobind-decorator';
+import { getMomentInstance } from 'app/shared/utils/Utils';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+import 'react-day-picker/lib/style.css';
+import moment from 'moment';
+import {
+  APP_LOCAL_DATETIME_FORMAT_Z_FORCE,
+  APP_LOCAL_DATE_FORMAT
+} from 'app/config/constants';
+import MomentLocaleUtils from 'react-day-picker/moment';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+
+type TokenExtendButtonProps = {
+  currentDate: string;
+  afterChangeDate: (newDate: string) => void;
+};
+
+@observer
+export default class TokenExtendButton extends React.Component<
+  TokenExtendButtonProps
+> {
+  @observable selectedDay: Date;
+
+  constructor(props: TokenExtendButtonProps) {
+    super(props);
+  }
+
+  @autobind
+  @action
+  handleDayChange(day: any) {
+    this.selectedDay = day;
+  }
+
+  @autobind
+  @action
+  handleSelectedDay() {
+    this.props.afterChangeDate(
+      moment(this.selectedDay).format(APP_LOCAL_DATETIME_FORMAT_Z_FORCE)
+    );
+  }
+
+  render() {
+    const extendSchedule = [
+      {
+        amount: 1,
+        idx: 1,
+        unit: 'week'
+      },
+      {
+        amount: 1,
+        idx: 2,
+        unit: 'month'
+      },
+      {
+        amount: 6,
+        idx: 3,
+        unit: 'months'
+      },
+      {
+        amount: 1,
+        idx: 4,
+        unit: 'year'
+      }
+    ];
+    return (
+      <DefaultTooltip
+        placement={'top'}
```

Comment:
This is the default, no need to specify.

Reply:
If don't specify placement, it will pop up on the left side. Is that ok?

Reply:
Hm that's weird. Let's keep it at the top. I took a look at the original default of the library. It must has been changed somewhere.